doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -28,60 +28,62 @@ The following chart illustrates basic development workflow:
            | pull request |         git push       \-------------/
            \--------------/
 
-Below we present an explanation of this chart. The explanation is written
-with the assumption that you, the reader, are a beginning developer who
-has an idea for a bugfix, but do not know exactly how to proceed. Watch
-the `Getting Started with Ceph Development
-<https://www.youtube.com/watch?v=t5UIehZ1oLs>`_ video for
-a practical summary of the same.
+The below explanation is written with the assumption that you, the reader, are
+a new contributor who has an idea for a bugfix or enhancement, but do not know
+exactly how to proceed. Watch the `Getting Started with Ceph Development
+<https://www.youtube.com/watch?v=t5UIehZ1oLs>`_ video for a practical summary
+of this workflow.
 
 Update the tracker
 ------------------
 
-Before you start, you should know the :ref:`issue-tracker` number of the bug
-you intend to fix. If there is no tracker issue, now is the time to create
-one.
+Before you start, you should know the :ref:`issue-tracker` (_Redmine_) number
```

Comment:
https://ceph--38144.org.readthedocs.build/en/38144/dev/developer_guide/basic-workflow/#update-the-tracker

_Redmine_, _documentation_, and _code_ render weird and should be rewritten.

Reply:
And this window italicizes anything between underscores, so that's nice to know.

Reply:
Italics were the intent.  The above render doesn't work for me fwiw.